src/views/catalog/customize/CustomizeVirtualMachine.tsx

```diff
@@ -33,8 +34,12 @@ const CustomizeVirtualMachine: React.FC = () => {
   });
 
   const Form = React.useMemo(() => {
+    const withCDSource = hasCustomizableCDSource(template);
     const withDiskSource = hasCustomizableDiskSource(template);
-    if (withDiskSource) {
+
+    if (withCDSource) {
```

Comment:
`if ... return ... else .. if` looks not nice
 can we find a nicer way to write it ?